apps/wpcom-block-editor/src/calypso/features/iframe-bridge-server.js

```diff
@@ -564,7 +564,7 @@ function handleInsertClassicBlockMedia( calypsoPort ) {
  * @param {MessagePort} calypsoPort Port used for communication with parent frame.
  */
 function handleCloseEditor( calypsoPort ) {
-	$( '#editor' ).on( 'click', '.edit-post-fullscreen-mode-close__toolbar a', e => {
+	$( '#editor' ).on( 'click', '.edit-post-header .edit-post-fullscreen-mode-close', e => {
```

Comment:
I think we should make sure this selector works for both old and new

Reply:
Yes, good catch! 🙌 
Fixed in b11651ed6c013a8993d7dc23b0794e548cbc5657